lib/packages.cf

```diff
@@ -72,7 +72,7 @@ bundle common package_module_knowledge
     slackware::
       "platform_default" string => "slackpkg";
 
-    windows::
+    windows.(cfengine_3_12|cfengine_3_13|cfengine_3_14|cfengine_3_15).!cfengien_3_12_0::
```

Comment:
Typo in the last class' name.

Reply:
FEK, i need to make emacs autocorrect that for me. I do it ALLL THE TIME

Reply:
pushed fix.